### ISSUE TITLE
Datamanagement

### ISSUE THREAD

**Issue by TLeenders** [VKGL-Kwaliteit/BioinformaticaVeldnorm#30]
This is something that popped into my mind recently, especially now that our field is delving into world of WGS.

There is a field standard for this (both in English and Dutch but god knows why these two are somewhat different: https://www.vkgl.nl/nl/kwaliteit/formulieren-documenten-kwaliteit/7-veldnormen). However, these standards are quite vague because they speak of "intermediate" results and "end" results.

What counts as what? they make some mentions of what counts as "end results", but there is still some debate for what counts as "intermediate" and if it should be kept or not. The Dutch version also mentions raw data, while the English version does not. 

Some other questions I still have:
how well protected should the data be kept? 

how and which validation data should be kept?

**Comment by hwmvandeutekom** [comment]
english version is older than the dutch version; so the latest was not translated.  

The document needs to be revised this year, anyone who wants to think about this norm from a bioinformatics point of view???  Good point about taking the big data into acount ;) 

protected <- patient data, so very well protected :)

**Comment by TLeenders** [comment]
Yes, I wouldn't mind doing this. Our lab has been "struggling" with data management, so I wouldn't mind delving further into the matter. We also discussed this last Friday and came to the conclusion that we as bioinformaticians should give a well informed advice regarding the retention of different types of patient data.
